locales/zh-cn.json

```
@@ -43,6 +43,8 @@
   "config.include_subfolders": "搜索子目录",
   "config.indent": "文案文件的缩进大小",
   "config.keep_fulfill": "始终用空字符串填充所有的键",
+  "config.key_prefix": "文案 key 的前缀",
+  "config.keygen_strategy": "文案 key 的生成策略, slug 默认策略, random 生成随机字符串",
```

```suggestion
  "config.keygen_strategy": "提取文案路径的生成策略",
```
Let keep the field description in the wiki :)